### PR TITLE
fix cashu invoice

### PR DESCRIPTION
**Pull request by callebtc**

### CODE REVIEW

File: katzensocks/cashu/cashu.go [hunk 1]
```diff
@@ -165,22 +166,23 @@ func (c *CashuApiClient) CreateInvoice(request InvoiceRequest) (*InvoiceResponse
...
-			}
-			if paymentStatus.Paid {
-				fmt.Println("Invoice paid!")
-				return
-			} else {
-				fmt.Println("Invoice not paid yet")
-			}
+	// go func() {
+	fmt.Printf("Checking invoice %s\n", response.CheckingId)
+	for {
```
Comment (mixmasala) [inline katzensocks/cashu/cashu.go:171]: does this make the API blocking now?

File: katzensocks/client/client.go [hunk 2]
```diff
@@ -145,8 +148,25 @@ func (c *Client) Topup(id []byte) chan error {
...
+		if err != nil {
+			c.log.Error("topup cashu, Balance Error:", err)
+			return
+		}
+		if balance.Balance < SATOSHIS_PER_REQUEST {
+			c.log.Infof("Balance too low: %d sats, creating Lightning invoice\n", balance.Balance)
+			invoice_request := cashu.InvoiceRequest{Amount: DEPOSIT_LIGHTNING_SATS}
+			resp, err := c.cashuClient.CreateInvoice(invoice_request)
+			if err != nil {
+				c.log.Error("topup cashu, CreateInvoice: %v", err)
```
Comment (mixmasala) [inline katzensocks/client/client.go:163]: I get a 400 error: 
10:56:03.516 ERRO katzensocks_client: topup cashu, CreateInvoice: 400 Bad Request

**Review by mixmasala — COMMENTED** [review]
see https://github.com/CloakedServices/CloakedNetworkPoC/issues/39

**Comment by callebtc** [comment]
Fixed after restart.

**Pull request merged**